tools/add_triage_signature.py

```diff
@@ -2327,6 +2327,67 @@ def _process_ticket(self, url, issue_key):
             self._update_triaging_ticket("Controller logs contain some warnings:\n" + f"{{code}}{warning_text}{{code}}")
 
 
+class MissingOSTreePivot(ErrorSignature):
+    """
+    This signature looks for missing pivot URL in rpm-ostree status of the control-plane hosts
+    """
+
+    PIVOT_URL_PATTERN = re.compile(r"pivot://.*")
+    CONTROL_PLANE_PATH = "*_bootstrap_*.tar.gz/logs_host_*/log-bundle-*.tar.gz/log-bundle-*/control-plane"
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(
+            *args,
```

Comment:
Just a though: are we running it only on masters or also on workers? 
control plane is only masters, right?

what will happen if a worker will suffer from similar issue? Lets not block this PR, but worth thinking about it

Reply:
Bootstrap doesn't gather logs about workers, so we have no information about them anyway

Reply:
We *might* have this information in a must-gather, but I'm not really sure of that

Reply:
I don't think we can get this info for workers.
Unless we update installer-gather code or scp files from the workers ourserlves.